gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem

```

Comment:
And this one

Reply:
I mentioned this in another comment on the review. I think VSCode on Mac didn't like the current line endings. I can just fix them up now since they're already on my machine